src/workerd/jsg/value.h

```diff
@@ -487,6 +499,18 @@ class StringWrapper {
 
     return kj::mv(result);
   }
+
+  kj::Maybe<USVString> tryUnwrap(v8::Local<v8::Context> context,
+      v8::Local<v8::Value> handle,
+      USVString*,
+      kj::Maybe<v8::Local<v8::Object>> parentObject) {
+    v8::Local<v8::String> str = check(handle->ToString(context));
+    v8::Isolate* isolate = context->GetIsolate();
+    auto buf = kj::heapArray<char>(str->Utf8LengthV2(isolate) + 1);
```

Comment:
technically we can check for str->ContainsOneByteOnly() and omit utf8 calls for one byte strings

Reply:
Let's look at that separately since that also holds true for several other places where we call `WriteUtf8...`